pkg/operator/kms/postcondition.go

```
@@ -0,0 +1,114 @@
+	return func(ctx context.Context, revision int32) error {
+		return validateKMSRevision(ctx, kubeClient, featureGateAccessor, revision)
+	}
+}
+
+func validateKMSRevision(ctx context.Context, kubeClient kubernetes.Interface, featureGateAccessor featuregates.FeatureGateAccess, revision int32) error {
+	if !featureGateAccessor.AreInitialFeatureGatesObserved() {
+		return nil
+	}
+	featureGates, err := featureGateAccessor.CurrentFeatureGates()
+	if err != nil {
+		return nil
```

golangci-lint (2.11.4)

[error] error is not nil (line 43) but it returns nil

(nilerr)